src/components/ebay-infotip/index.marko

```diff
@@ -63,21 +68,23 @@ $ var pointer = input.pointer || "bottom";
                     heading=input.heading
                     content=input.content
                     a11y-close-text=input.a11yCloseButtonText
-                    onOverlay-close("handleOverlayClose")/>
+                    onOverlay-close("handleOverlayClose")
+                />
             </if>
-            <else>
-                <ebay-drawer-dialog
-                    open=state.open
-                    a11y-close-text=input.a11yCloseButtonText
-                    on-open("handleExpand")
-                    on-close("handleCollapse")
-                    aria-label=input.ariaLabel>
-                    <@header>
-                        <${input.heading && input.heading.renderBody}/>
-                    </@header>
-                    <${input.content}/>
-                </ebay-drawer-dialog>
-            </else>
         </span>
     </ebay-tooltip-base>
 </span>
+<if(isModal === true)>
+    <ebay-lightbox-dialog
+        open=state.open
+        a11y-close-text=input.a11yCloseButtonText
+        on-open("handleExpand")
+        on-close("handleCollapse")
+        aria-label=input.ariaLabel
+    >
+        <@header>
+            <${input.heading && input.heading.renderBody}/>
```

Comment:
I don't think this extra `&&` is necessary since `renderBody` is implied for dynamic tag, unless that is not a feature in Marko 4 or something